alibi/explainers/tests/test_simiarlity/conftest.py

```diff
@@ -143,18 +144,26 @@ def linear_models(request):
     return tf_model, tf_loss, torch_model, torch_loss
 
 
-def tf_linear_model(input_shape, output_shape):
+def tf_linear_model(input_shape, output_shape, batch_norm=False):
     """
     Constructs a linear model for `tensorflow`.
     """
-    return keras.Sequential([
-        keras.layers.InputLayer(input_shape=input_shape),
-        keras.layers.Dense(output_shape),
-        keras.layers.Softmax()
+    layers = [
+        tf.keras.layers.InputLayer(input_shape=input_shape),
+        tf.keras.layers.Flatten(),
```

Comment:
Assume this was added intentionally?

Reply:
Sort of, changes were made that have now been reverted. I think I left the above. I've now reverted it also.